app/src/main/java/com/hedvig/app/feature/home/ui/HomeModel.kt

```diff
@@ -36,6 +36,10 @@ sealed class HomeModel {
         object Terminated : BodyText()
     }
 
+    data class ClaimStatus(
+        val claims: List<HomeQuery.ClaimStatus>
```

Comment:
A recent, but arguably good, practice we've started following in the app is that we remap from network-layer types to our own types. I know that this is not what we're currently doing on this screen, but I think we should do it here as well (i.e create our own `Claim`-type that we use here, or something of the sort) 🙂

Reply:
Yeap, absolutely reasonable